models/style.go

```diff
@@ -223,6 +224,20 @@ func (s *APIStyle) GetSourceCodeCRC32() string {
 	return fmt.Sprintf("%08x", crc32.ChecksumIEEE([]byte(s.Code)))
 }
 
+func AbleToReview(uid, sid uint) (bool, string) {
+	reviewSpam := new(Review)
+	// Collecting of the error is not needed.
+	// As we simply check "valid" data by checking if ID is a positive integer.
+	if _ = reviewSpam.FindLastFromUser(sid, uid); reviewSpam.ID > 0 {
+		log.Info.Printf("User %d tried to review style %v more than once.\n", uid, sid)
+		week := -7 * 24 * time.Hour
+		if reviewSpam.CreatedAt.After(time.Now().Add(week)) {
+			return false, time.Since(reviewSpam.CreatedAt.Add(week)).Truncate(time.Second).String()
+		}
```

Comment:
Similar to my previous suggestion, just better.

```suggestion
		t := time.Now().Sub(reviewSpam.CreatedAt)
		if t < 7*24*time.Hour {
			t = 7*24*time.Hour - t
			return false, t.Round(time.Second).String()
		}
```